cpp/src/slice2js/Gen.cpp

```diff
@@ -241,7 +241,7 @@ namespace
     }
 }
 
-Slice::JsVisitor::JsVisitor(Output& out, const vector<pair<string, string>>& imports) : _out(out), _imports(imports) {}
```

Comment:
We never passed `imports` to the constructor, it was always default-initialized.

Reply:
Nevermind, it turns out that this `_imports` thing was completely dead code.
So I just completely removed it instead...